wayback/_client.py

```diff
@@ -611,7 +626,8 @@ def search(self, url, *, match_type=None, limit=1000, offset=None,
 
     def get_memento(self, url, datetime=None, mode=Mode.original, *,
                     exact=True, exact_redirects=None,
-                    target_window=24 * 60 * 60, follow_redirects=True):
+                    target_window=24 * 60 * 60, follow_redirects=True,
+                    calls_per_second=None):
```

Comment:
@LionSzl same as with `search()`: do you have a use-case that requires configuration on each call, rather than differently configured clients? I’d prefer not to have this in order to keep things simpler.